### PR TITLE
Remove unused vars 2

### PR DESCRIPTION
**Pull request by ger-benjamin**
For https://jira.camptocamp.com/browse/GSGMF-261

Still to fix (in a next PR): 
 - One require for injection,
 - Requires in the tests files,
 - Some require in apps

~50 / 300+ warnings left

### CODE REVIEW

**Comment by ger-benjamin** [comment]
To debug the tests, I went back on the use of EPSG21781 as variable in the tests. Because I still have work on tests and I'll see this problem later.

File: test/spec/services/autoprojection.spec.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ describe('ngeo.misc.AutoProjection', () => {
     const coordinatesA = [600000, 200000];
     const coordinatesB = [8, 47];
     const viewProjection = ol.proj.get('EPSG:21781');
+    debugger;
```
Comment (sbrunner) [inline test/spec/services/autoprojection.spec.js]: ?

Comment (ger-benjamin) [reply test/spec/services/autoprojection.spec.js]: grrr

Comment (ger-benjamin) [reply test/spec/services/autoprojection.spec.js]: thanks

**Pull request merged**